<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_titulo">
      <property name="text">
       <string>Gerador de Senhas</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_senha">
        <property name="text">
         <string>Senha:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linha_senha"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_slider">
        <property name="text">
         <string>Comprimento:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="slider_comprimento">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_comprimento">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupbox_caracteres">
      <property name="title">
       <string>Caracteres</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkbox_minusculas">
         <property name="text">
          <string>Minúsculas</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkbox_numeros">
         <property name="text">
          <string>Números</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkbox_maiusculas">
         <property name="text">
          <string>Maiúsculas</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkbox_especiais">
         <property name="text">
          <string>Especiais</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="botao_gerar">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Gerar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
